docs/getting-started/setting-up-git.mdx

```diff
@@ -10,7 +10,7 @@ description: "There are a number of ways to enable version control, collaboratio
 
 ## Choose your adventure
 
-There three different ways to use Git in Mage and _four_ ways to version control your files— we'll discussing each of them.
+There three different ways to use Git in Mage and _four_ ways to version control your files— we'll discussing each of them. We should note that the thre ways of using Git are mutually exclusive. That means that you can only choose one of the three options. However, you can use any of the four version control methods in conjunction with any of the three Git methods.
```

Comment:
What do you mean by "use any of the four version control methods in conjunction with any of the three Git methods."

Reply:
Mage "version control" can be used no matter what

Reply:
But you cant use git sync + other git functionality

Reply:
we usually don't use git sync + other git functionality since it cause conflicts.
cc: @dy46 am I right?

Reply:
yeah, we should recommend users to use git sync separately.

Reply:
That's exactly what I'm saying

Reply:
@mattppal version control and git sync are not compatible. i think it makes sense to just say that the git features are all mutually exclusive in the documentation.